src/main.js

```diff
@@ -46,6 +47,7 @@ function toggleStarOnData() {
       savedIdeas[i].star = true;
     }
   }
+  saveIdeaToStorage()
```

Comment:
Updates the savedIdeas array and saves this new array to localStorage, overwriting what is currently saved there.